src/main/java/org/broadinstitute/hellbender/utils/collections/NestedIntegerArray.java

```diff
@@ -83,6 +83,81 @@ public T get(final int... keys) {
     }
 
     /**
+     * Specialized version of get for 2 parameters.
+     * Varagrs have a large cost because the arg array is allocated everytime.
+     * Using a specialized method eliminates that performance problem.
+     */
+    @SuppressWarnings("unchecked")
+    public T get2(final int key0, final int key1) {
```

Comment:
Note: I use a different way of doing this in https://github.com/broadinstitute/gatk/commit/6bd9f599d62452f92fceb3fd252a85b141d14b6c
(that way is probably preferable)